e2e/ephemeral_check.go

```diff
@@ -0,0 +1,260 @@
+package e2e
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+
+	snapapi "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+//go:embed testdata/ephemeral_tests/pod-ephemeral-volume-device.yaml
+var podEphemeralBlockYAMLTemplate string
+
+//go:embed testdata/ephemeral_tests/pod-ephemeral-volume-mount.yaml
+var podEphemeralFSYAMLTemplate string
+
+//go:embed testdata/ephemeral_tests/ephemeral-volume-snapshot.yaml
+var ephemeralVolumeSnapshotYAMLTemplate string
+
+//go:embed testdata/ephemeral_tests/ephemeral-clone.yaml
+var ephemeralPvcCloneYAMLTemplate string
+
+//go:embed testdata/ephemeral_tests/ephemeral-snapshot-restore.yaml
+var ephemeralPvcSnapshotRestoreYAMLTemplate string
+
+//go:embed testdata/ephemeral_tests/pod-volume-mount-template.yaml
+var podFSYAMLTemplate string
+
+//go:embed testdata/ephemeral_tests/pod-volume-device-template.yaml
+var podBlockYAMLTemplate string
```

Comment:
Instead of multiple variable declarations, create a `var` block?

Reply:
Thanks, updated.